src/components/cards/SettingsCard.tsx

```diff
@@ -112,13 +114,15 @@ const SettingsCard: React.FC<Props> = ({ user }) => {
             </Text>
           </div>
           <div className={classes.tagContainer}>
-            <Tag
-              background="#4E555C"
-              color="#ffffff"
-              style={{ marginLeft: '10px' }}
-            >
-              {getUserType(user)}
-            </Tag>
+            {Object.values(userTags).map((tag) => (
```

Comment:
What's the purpose of `userTags` being an object/dictionary while only using them as an array?

Reply:
so the thing is, when we tried to search, if it is possible to return an array, it was shown that it is not possible, and so we used an object for only that reason
reference: "https://stackoverflow.com/questions/12059478/return-array-from-function"

Reply:
Since `userTags` are an array now you can directly map from that array and omit this `Object.values` construction.

Reply:
@lsc0498 still for some reason, it's not working, so what i mean is, it is still considered to be an object, and when i try to cast it to an array which is compatible, and use the map function it concatenates all of them, and show all the tags in one tag

Reply:
and also shouldi merge the branch??

Reply:
Just use `userTags.map`. That should actually work.

Reply:
change made